context/app/static/js/components/entity-search/Search/RequestTransporter.js

```diff
@@ -0,0 +1,20 @@
+// Copied from https://www.searchkit.co/docs/reference/searchkit-sdk#network-transport
+// Licensed under https://github.com/searchkit/searchkit/blob/next/LICENCE
```

Comment:
Thanks! I think the Apache license requires two more things from us:
- Include the original copyright
- Include the original license

Maybe just paste both into this file? Or maybe reorganize so all the code we're copy-and-pasting from them lives in the same directory?

Reply:
I moved the modified files to a directory and included copies of the license and copyright.